sunpy/database/caching.py

```diff
@@ -1,36 +1,37 @@
-# Author: Simon Liedtke <liedtke.simon@googlemail.com>
-#
-# This module was developed with funding provided by
-# the Google Summer of Code (2013).
+# This module was developed with funding provided by the Google Summer of Code (2013).
```

Comment:
General question here, do we want to keep these comments about GSOC? We have a doc section on this don't we?

Reply:
We planned to have one? I didn't think we have one but I ain't checked.

Reply:
We did have one and I moved it all there.